corehq/apps/receiverwrapper/repeater_generators.py

```diff
@@ -37,6 +37,13 @@ def get_payload(self, repeat_record, payload_doc):
         return payload_doc.to_xml(self.repeater.version or V2, include_case_on_closed=True)
 
 
+@RegisterGenerator(CaseRepeater, 'case_json', 'JSON', is_default=False)
+class CaseRepeaterJsonPayloadGenerator(BasePayloadGenerator):
+    def get_payload(self, repeat_record, payload_doc):
+        del payload_doc['actions']
+        return json.dumps(payload_doc.to_json())
```

Comment:
Should we not produce the same format as the API which calls `case.get_json()`: https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/case/models.py#L232-232